src/figuresClasses.ts

```diff
@@ -1,19 +1,90 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
+
```

Comment:
Remove empty lines between fields.

Reply:
Check other similar cases.

Reply:
@OleziO Excuse me, but my linter asks me to write this way (Expected blank line between class members) Mb i should write properties somehow else?